nixos/modules/virtualisation/xen-dom0.nix

```
@@ -692,17 +748,28 @@ in
       '';
 
       # Xen Bootspec extension. This extension allows NixOS bootloaders to
-      # fetch the `xen.efi` path and access the `cfg.bootParams` option.
+      # fetch the dom0 kernel paths and access the `cfg.boot.params` option.
       bootspec.extensions = {
+        # Bootspec extension v1 is deprecated, and will be removed in 25.11
```

```suggestion
        # Bootspec extension v1 is deprecated, and will be removed in 26.05
```
We're in 25.11 already.

---

Well, that was fast! I'm still rebasing and fixing a few rebase issues, as well as re-testing this - but I'll be sure to bump these versions.